src/UrlMatcher.php

```diff
@@ -267,19 +264,15 @@ static function ($allowedMethods, Route $route) use ($path) {
         return MatchingResult::fromFailure($allowedMethods);
     }
 
-    private function matchHost(Route $route): bool
-    {
-        return $this->request->getUri()->getHost() === $route->getHost();
-    }
-
     /**
      * Inject routes into the underlying router
      */
     private function injectRoutes(): void
     {
         foreach ($this->routeCollection->getRoutes() as $index => $route) {
             /** @var Route $route */
-            $this->fastRouteCollector->addRoute($route->getMethods(), $route->getPattern(), $route->getName());
+            $hostPattern = $route->getHost() ?? '{_host:.*}';
```

Comment:
You should do it in the https://github.com/yiisoft/router/blob/master/src/RouteCollection.php

Reply:
> You should do it in the https://github.com/yiisoft/router/blob/master/src/RouteCollection.php

Everything is ok, it's my bad